examples/taskruns/sidecar-interp.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  generateName: sidecar-interp-
+spec:
+  taskSpec:
+    inputs:
+      params:
+        - name: some-input
+          default: foo
+    volumes:
+      - name: shared
+        emptyDir: {}
+    sidecars:
+      - name: value-sidecar
+        image: ubuntu
+        command:
+          - /bin/bash
+        args:
+          - -c
+          - "echo $(inputs.params.some-input) > /shared/value && sleep infinity"
+        volumeMounts:
+          - name: shared
+            mountPath: /shared
+    steps:
+      - name: check-value
+        image: ubuntu
+        script: |
+          #!/bin/bash
+          VALUE=$(cat /shared/value)
+          if [ $VALUE == 'foo' ]; then
```

Comment:
This can just be `[[ $VALUE == 'foo' ]]`

Otherwise it's the equivalent of:

```go
if x == "foo" {
  return true
} else {
  return false
}
```

vs 

```go
return x == "foo"
```

Reply:
Fixed